### PR TITLE
Removing detection rules from the `problemchild` package

### PR DESCRIPTION
**Pull request by ajosh0504**
## What does this PR do?
This PR moves detection rules associated with DGA (dga) out of the integration package. 

## Checklist
- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
- Spin up a local Elasticsearch cluster
- Verify that the package version shown is 2.0.0, Subscription is `platinum` and License is `Elastic-2.0`
- Install the Living off the Land Detection assets
- Verify that no detection rules with the `Living off the Land` tag show up on the Rules Management page under Security
- Verify that `-embedding` does not appear on the blocklist processor in the inference pipeline

## Related Issues
Not linking since these are internal issues

## Screenshots
-  License and subscription info is as expected
Screen Shot 2023-09-21 at 10 22 09 AM

-  No rules appear upon installation of package assets
Screen Shot 2023-09-21 at 10 22 44 AM

-  `-embedding` is not part of the blocklist in the inference processor
Screen Shot 2023-09-21 at 10 23 56 AM

### CODE REVIEW

File: packages/problemchild/docs/README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+*_Note_**: v2.0.0 of the package introduces breaking changes, namely updating ML job IDs and deprecating detection rules from the package. To continue recieving updates to LotL Detection, we recommend uninstalling existing ML jobs and rules associated with this package, upgrading to v2.0.0, and installing the new rules as described in the [Enable detection rules](#enable-detection-rules) section below.
 
 ### (Required) Set up the ingest pipeline
 
 Once you’ve installed the package you can ingest your data using the ingest pipeline. This will enrich your incoming data with its predictions from the machine learning model.
 
-This pipeline is designed to work with [Winlogbeat data](https://www.elastic.co/downloads/beats/winlogbeat).
+This pipeline is designed to work with Winlogbeat and Endpoint data.
 
 ### (Optional) Add preconfigured anomaly detection jobs
```
Comment (sodhikirti07) [inline packages/problemchild/docs/README.md:21]: Might want to remove these `Required` and `Optional` labels. We decided to not keep them in the new packages/package updates as `ded`, `lmd` , etc.

Comment (ajosh0504) [reply packages/problemchild/docs/README.md:21]: Done!

File: packages/problemchild/docs/README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
 
-Follow these instructions to ingest data with the ingest pipeline and enrich your indices with inference data. Then use these detection rules and anomaly detection jobs to detect LotL attacks. For more detailed information refer to [this](https://www.elastic.co/security-labs/detecting-living-off-the-land-attacks-with-new-elastic-integration) blog.
+Follow these instructions to ingest data with the ingest pipeline and enrich your indices with inference data. Then use the anomaly detection jobs in this package and associated rules in the Detection Engine, to detect LotL attacks. For more detailed information refer to [this](https://www.elastic.co/security-labs/detecting-living-off-the-land-attacks-with-new-elastic-integration) blog.
+
+*_Note_**: v2.0.0 of the package introduces breaking changes, namely updating ML job IDs and deprecating detection rules from the package. To continue recieving updates to LotL Detection, we recommend uninstalling existing ML jobs and rules associated with this package, upgrading to v2.0.0, and installing the new rules as described in the [Enable detection rules](#enable-detection-rules) section below.
```
[the remark this answers is not in the document]
Comment (ajosh0504) [reply packages/problemchild/docs/README.md]: Nice catch!

**Comment by qn895** [comment]
Tested latest changes and license change LGTM.

On Platinum license - Install ProblemChild 2.0.0 on Platinum license successfully ✅
On Platinum license - Install ProblemChild 1.0.0 and then upgrade to 2.0.0 successfully ✅
On Basic license - Install ProblemChild 2.0.0 and get an error as expected (error on incompatible license) ✅
On Basic license - Install ProblemChild 1.0.0 and get an error as expected (error due to non-compliant with ML) ✅

**Review by peteharverson — APPROVED** [review]
Approving for ML after seeing the https://github.com/elastic/integrations/pull/7903#issuecomment-1735978533 from @qn895

File: packages/problemchild/manifest.yml [hunk 3]
```diff
@@ -11,6 +12,8 @@ categories:
 release: ga
 conditions:
   kibana.version: "^8.0.0"
+  elastic:
+    subscription: platinum
```
[the remark this answers is not in the document]
Comment (ajosh0504) [reply packages/problemchild/manifest.yml:16]: @qn895 Could you please comment with results from your testing? Thank you!

**Pull request merged**